labonneboite/common/search.py

```diff
@@ -427,8 +427,12 @@ def get_companies_from_es_and_db(json_body, sort):
         company_objects = Office.query.filter(Office.siret.in_(siret_list))
         company_dict = {}
 
+        # FIXME it's not great to add new properties to an existing object. It
+        # would be better to wrap the office objects in a new OfficeResult
+        # class that would add new properties related to the query.
         for obj in company_objects:
             # Get the corresponding item from the Elasticsearch results.
+            # FIXME this is very inefficient, as we iterate over all es results for every company
             es_company = next((item for item in res['hits']['hits'] if item["_source"]["siret"] == obj.siret))
```

Comment:
true, but it's just 10 objects right?  (10 companies per page)

Reply:
Hmmm oui c'est juste, j'avais pas réalisé.